src/main/java/digital/slovensko/autogram/ui/gui/SigningDialogController.java

```diff
@@ -73,6 +82,41 @@ public void onChangeKeyButtonPressed(ActionEvent event) {
         autogram.pickSigningKeyAndThen(gui::setActiveSigningKey);
     }
 
+    public void onShowPresentSignaturesButtonPressed(ActionEvent event) {
```

Comment:
Ten nazov controllera mi pride zvlastny. `SignaturesController` nie je lepsi nazov?

Reply:
Ak niekde budeme pridavat vizualny podpis, tak sa to asi bude diat priamo v tom podpisovacom controlleri, cize toto asi nevadi. Premenujme.